netlogo-core/src/test/fileformat/ModelSectionTest.scala

```diff
@@ -51,6 +51,15 @@ trait ModelSectionTest[A, B <: ModelFormat[A, B], C] extends FunSuite {
     }
   }
 
+  def testErrorsOnDeserializationAndWrongArity(
+    description: String, serializedVersion: A, error: String): Unit = {
+    test(s"errors when deserializing $description") {
+      val s = subject
+      val m = s.deserialize(serializedVersion)(new Model())
+      assert(!m.isFailure)
+    }
+  }
+
```

Comment:
This is called `testErrorsOnDeserializationAndWrongArity`, so the `testErrors...` part makes me think the result should be an error, but the `assert(!m.isFailure)` is checking success.  Would a better name be something like `testNoErrors...`?